Source/WebInspectorUI/UserInterface/Views/FindBanner.js

```diff
@@ -44,12 +44,14 @@ WI.FindBanner = class FindBanner extends WI.NavigationItem
         this._inputField.type = "search";
         this._inputField.placeholder = " "; // This is necessary for :placeholder-shown.
         this._inputField.spellcheck = false;
-        this._inputField.incremental = true;
         this._inputField.setAttribute("results", 5);
         this._inputField.setAttribute("autosave", "inspector-search");
         this._inputField.addEventListener("keydown", this._inputFieldKeyDown.bind(this), false);
         this._inputField.addEventListener("keyup", this._inputFieldKeyUp.bind(this), false);
-        this._inputField.addEventListener("search", this._inputFieldSearch.bind(this), false);
+        this._inputFieldInputThrottler = new Throttler(this._inputFieldInput.bind(this), 250);
+        this._inputField.addEventListener("input", (event) => {
+            this._inputFieldInputThrottler.fire();
```

Comment:
we should pass along the most recent `event` just in case `_inputFieldInput` needs to use it
```
this._inputFieldInputThrottler.fire(event);
```

Reply:
Fixed in pr.